listenbrainz/webserver/views/art_api.py

```diff
@@ -287,6 +287,23 @@ def _cover_art_yim_artists(user_name, stats):
     )
 
 
+def _cover_art_yim_playlist(user_name, stats, key):
+    """ Create the SVG using playlist tracks' cover arts for the given YIM playlist. """
+    all_cover_arts = stats[f"{key}-coverart"]
+    image_urls = []
+    for track in stats[key]["track"]:
```

Comment:
In my case,  there are lots of repetitions of cover art for the image, which is not ideal:
<img width="475" alt="image" src="https://user-images.githubusercontent.com/6179856/209698418-c808e37a-cbb4-40cc-93c7-af0b41b0cd57.png">

Can this be deduped, rather than just going down the list of tracks in order?

Reply:
Yes can do.